src/Microsoft.ML.FastTree/FastTreeStatic.cs

```diff
@@ -105,6 +105,48 @@ public static class FastTreeStatic
             return rec.Output;
         }
 
+        /// <summary>
+        /// FastTree <see cref="RankerContext"/>.
+        /// </summary>
+        /// <param name="ctx">The <see cref="RegressionContext"/>.</param>
+        /// <param name="label">The label column.</param>
+        /// <param name="features">The features colum.</param>
+        /// <param name="groupId">The name of the groupId column.</param>
+        /// <param name="weights">The weights column.</param>
+        /// <param name="numLeaves">The number of leaves to use.</param>
```

Comment:
>to use [](start = 57, length = 6)

"Leaves to use" is kind of weight. Maybe "maximum number of leaves per decision tree" would be better. I might also prefer to have number of *trees* above number of *leaves* (that is, reverse the two args).